app/src/main/kotlin/com/saltedge/authenticator/models/repository/PreferenceRepositoryAbs.kt

```diff
@@ -34,6 +34,7 @@ interface PreferenceRepositoryAbs {
     var pinInputAttempts: Int
     var blockPinInputTillTime: Long
     var screenshotLockEnabled: Boolean
+    var systemDarkMode: Boolean
```

Comment:
i think it is necessary to unify next variable names `nightMode` and `systemDarkMode`